src/components/layout/Container.tsx

```diff
@@ -26,13 +26,13 @@ export const Container: React.FC<ContainerProps> = ({
       style={{
         ...(padding
           ? {
-              paddingLeft: isMobile ? "10%" : "30%",
-              paddingRight: isMobile ? "10%" : "30%",
+              paddingLeft: isMobile ? "10%" : "10%",
+              paddingRight: isMobile ? "10%" : "10%",
```

Comment:
à quoi sert le isMobile si c'est la meme taille ? 
```suggestion
              paddingLeft: "10%",
              paddingRight: "10%",
```